.editorconfig

```diff
@@ -4,12 +4,18 @@ end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
 
+[*.md]
+indent_style = space
+indent_size = 4
```

Comment:
are you sure about this because i thought you were also accustomed to (the I think much more convenient) `2` spaces (vgl. https://github.com/delb-xml/delb-py/blob/integration-tests-location-path/ROADMAP.md?plain=1#L9 )?

Reply:
fair point, i had only thought of code blocks.